src/Page/Guide/View.elm

```diff
@@ -267,6 +281,13 @@ viewPrintGuide language =
         ]
 
 
+viewHeaderIcon : String -> Html Msg
+viewHeaderIcon url =
+    span [ attribute "aria-hidden" "true" ]
+        [ img [ src url, css [ display inlineBlock, marginRight (rem 0.5), maxWidth (em 1) ] ] []
```

Comment:
Can we extract the styles to a function to fit with the pattern used elsewhere? I think it's OK if it's a single attribute but elsewhere when we have lists, it's abstracted to make the markup more parseable by our brains.

Reply:
can and have